### DISCUSSION TITLE
Default has changed from original version

### DISCUSSION THREAD

**Discussion by podfeet** [bartificer/xkpasswd-js#79] [General]
According to the original documentation, the default configuration preset should look in this form:

`12:settle:SUCCEED:summer:48`

But in the current beta, a typical default password looks like this:

`~~19%Foreign%Heavy%Believe%57~~`

I'm working on the user documentation so I need to know if this is by design or if it might change back to the original default for the docs.

**Comment by podfeet** [comment]
Same with WEB16. Original documentation:

`:baby.ohio.DEAR:`

Beta:

`pure*iron*wide*34`

**Comment by hepabolu** [comment]
I used the settings as found in the Perl code. I prefer to stick to these settings. Of course the implementation (i.e. the generation of the password sequences) might be wrong, but these I also compared the code with the Perl code and tested the examples I found here and there in the code's documentation.

So unless someone can prove that it's implemented wrong, I prefer to stick to these settings as the new default.
